circles-app/src/routes/api/price/+server.ts

```
@@ -62,7 +62,7 @@ export const GET: RequestHandler = async ({ url }) => {
 
     const history = rows.map(r => ({
       timestamp: new Date(r.bucket),
-      price: staticCirclesToCircles(parseFloat(r.price))
+      price: CirclesConverter.attoStaticCirclesToAttoCircles(CirclesConverter.circlesToAttoCircles(parseFloat(r.price)))
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)